### PR TITLE
Add Ecosia to "In the Wild" section.

### PR DESCRIPTION
**Pull request by gmccue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2136,6 +2136,7 @@ Other Style Guides
   - **Compass Learning**: [compasslearning/javascript-style-guide](https://github.com/compasslearning/javascript-style-guide)
   - **DailyMotion**: [dailymotion/javascript](https://github.com/dailymotion/javascript)
   - **Digitpaint** [digitpaint/javascript](https://github.com/digitpaint/javascript)
+  - **Ecosia**: [evernote/javascript](https://github.com/ecosia/javascript)
```
Comment (justjake) [inline README.md]: You want the text to be "evernote/JavaScript"?

Comment (gmccue) [inline README.md]: Wow, my bad! Thanks for catching that @justjake

**Comment by hshoff** [comment]
Awesome, thanks @gmccue! Ecosia does great work. I think I just planted 3 trees. :deciduous_tree:

**Comment by gmccue** [comment]
Excellent! Thanks for the kind words & merge, @hshoff!

**Pull request merged**